src/thor/bidirectional_astar.cc

```diff
@@ -1329,17 +1337,6 @@ std::vector<std::vector<PathInfo>> BidirectionalAStar::FormPath(GraphReader& gra
   return paths;
 }
 
-void BidirectionalAStar::ModifyHierarchyLimits() {
```

Comment:
since this is just a static division we just store those modified values as the defaults in the config and remove this